app/src/main/java/fr/free/nrw/commons/nearby/NearbyMapFragment.java

```diff
@@ -514,16 +519,18 @@ public void onSlide(@NonNull View bottomSheet, float slideOffset) {
      */
     private void setupMapView(Bundle savedInstanceState) {
         Timber.d("setupMapView called");
+
         MapboxMapOptions options = new MapboxMapOptions()
```

Comment:
It seems like the only change here is indentation. Please try to just change he issue related codes. Other style fixes makes it hard to review. This style issues can be fixed on another PR.

Reply:
Sorry for that . Will keep in mind from next time.